src/util/dstring.h

```diff
@@ -158,6 +158,12 @@ inline size_t hash_string(const dstringt &s)
   return s.hash();
 }
 
+inline size_t const_hash_string(const dstringt &s)
+{
+  return hash_string(string_container.get_string(s.get_no()));
+}
```

Comment:
I see, thanks for clarifying. May I ask for such information to be included in commit messages for others to understand what's happening - that would be very helpful. Also, the `const` in the name got me very confused; how about "stable"?